SU2_CFD/include/numerics/flow/flow_sources.hpp

```diff
@@ -323,21 +341,83 @@ class CSourceWindGust final : public CSourceBase_Flow {
 };
 
 /*!
- * \class CSourceRadiation
- * \brief Class for a source term due to radiation.
+ * \class CSourceIncStreamwise_Periodic
+ * \brief Class for the source term integration of a streamwise periodic body force in the incompressible solver.
  * \ingroup SourceDiscr
- * \author Ruben Sanchez
+ * \author T. Kattmann
  */
-class CSourceRadiation : public CSourceBase_Flow {
+class CSourceIncStreamwise_Periodic final : public CSourceBase_Flow {
 private:
-  bool implicit;
 
+  bool turbulent; /*!< \brief Turbulence model used. */
+  bool energy;    /*!< \brief Energy equation on. */
+  bool streamwisePeriodic_temperature; /*!< \brief Periodicity in energy equation */
+  su2double Streamwise_Coord_Vector[MAXNDIM] = {0.0}; /*!< \brief Translation vector between streamwise periodic surfaces. */
```

Comment:
It is hard to choose one style of formatting given the variety in SU2 xD

Reply:
Yeah, and I think I changed that back and forth a couple of times ... I can change that to a more consistent style if that aligns better w/ sth else

Reply:
Nah... I was just messing, if there is a clear style in the file I'm working on I try to follow it, otherwise... FREEEEDOMMMM